testgrid/config/config.yaml

```diff
@@ -1611,6 +1620,17 @@ dashboards:
   - name: gci-1.4-gci-latest-upgrade-master
     test_group_name: kubernetes-e2e-gke-gci-1.4-gci-latest-upgrade-master
 
+- name: latest-upgrades
```

Comment:
Yea... there should be some cleanup/consolidation of various "latest" upgrades. I will get around to it... this is just more important right now.